docs/release/release-tasks.md

```diff
@@ -513,3 +516,28 @@ Publish the release.
 
    1. The draft release should be automatically created via the [Create Release GitHub Action](https://github.com/kubernetes-sigs/cluster-api/actions/workflows/release.yml) with release notes previously committed to the repo by the release team.
    2. Publish the release. Ensure release is flagged as `pre-release` for all `beta` and `rc` releases or `latest` for a new release in the most recent release branch.
+
+## Suggestions for Team Leads
+  * Clearly communicate with the team that the majority of the work during the release cycle will be a collaborative effort.
+  * Establish an ownership rotation policy in consultation with respective team members.
+  * Provide opportunities for team members to take the lead in cutting a release within the cycle, based on feasibility.
+  * Define backup ownership and tasks for team members, such as:
+      * Hosting community meetings.
+      * Communicating key updates during office hours meetings when necessary.
+      * Sharing release-related updates with the CAPI community.
+      * Monitoring and reporting CI status regularly to the release team.
+      * Scheduling additional meetings as required to facilitate a smooth release cycle.
+
+### Time Commitments
+Here's a rough estimate of the time requirements per week:
```

Comment:
Just wondering how this was estimated. I'm sensitive to there being some amount of false precision here, and I don't want to scare folks off from volunteering. 

16 hours a week is a lot of time for folks. I find it hard to estimate based of my own experience given I'm working on CAPI even when not specifically doing release tasks.

Reply:
@killianmuldoon agreed these were just initial rough estimates and we intended to get more accurate inputs from here, I will scope it to 6-12. Hope thats okay